testsuite/single_node_performance.py

```diff
@@ -275,6 +275,10 @@ class RunGroupConfig:
     RunGroupConfig(key=RunGroupKey("order-book-balanced-size-skewed80-pct50-markets"), included_in=Flow.ORDER_BOOK | Flow.CONTINUOUS, waived=True),
     RunGroupConfig(key=RunGroupKey("monotonic-counter-single"), included_in=Flow.CONTINUOUS, waived=True),
 
+    RunGroupConfig(key=RunGroupKey("fibonacci-recursive-10"), included_in=Flow.CONTINUOUS),
```

Comment:
If you have not calibrated numbers, probably add `, waived=True` for now so CI ignores numbers for these